### ISSUE TITLE
More accessibility advice

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/coach#11]
Here's a list of some accessibility advice we should implement. Let's break it down and create issues for the most important ones:
http://aduggin.github.io/accessibility-fails/

**Comment by softwareklinic** [comment]
Is the accessibility check based on WCAG 2.0 standards... or rather what are the accessibility standards being incorporated..

**Comment by soulgalore** [comment]
We actually started out with the most important stuff from this post: https://www.marcozehe.de/2015/12/14/the-web-accessibility-basics/ 

Just to have an easy start. Implementing the full WCAG 2.0 guidelines would be cool but I think we need a lot of help 2do that.

**Comment by softwareklinic** [comment]
We should collaborate with the developer of Asqatasun – its an Open source project on github – to see how these to projects can benefit from each other…

**Comment by soulgalore** [comment]
Let us create a new issue with more updated list of what should be done.
